### DISCUSSION TITLE
Our new logo

### DISCUSSION THREAD

**Discussion by becarpenter** [becarpenter/book6#26] [Announcements]
I thought this project needed a logo, so I used all my ![skills and imagination](../blob/main/book6logo.png). I did check it at a color-blindness site and it should be OK for most people.

Please review all the material already in the book, and raise issues or PRs wherever you see a problem or something missing. There are lots of places where we still need more text: don't hesitate. Some case studies would be **great** and don't need to be very long, especially if there's an existing description on the web. Just [point to it](https://help.uis.cam.ac.uk/service/network-services/ip/ipv6-at-cambridge).

Start writing! Tell all your friends to start writing! The instructions are at [CONTRIBUTING](https://github.com/becarpenter/book6/blob/main/CONTRIBUTING.md).

**Comment by joaolucasmacedo** [comment]
I've made a logo. Check it out:
![Capa](https://user-images.githubusercontent.com/66760118/203323209-fab1e16a-042a-4cb5-bb5a-a650b32ed597.svg)

**Reply by becarpenter** [reply]
:-) but a bit too detailed for use as an icon.
